platform/features/table/src/controllers/MCTTableController.js

```diff
@@ -400,74 +414,50 @@ define(
          * pre-calculate optimal column sizes without having to render
          * every row.
          */
-        MCTTableController.prototype.findLargestRow = function (rows) {
-            var largestRow = rows.reduce(function (largestRow, row) {
+        MCTTableController.prototype.buildLargestRow = function (rows) {
+            var largestRow = rows.reduce(function (prevLargest, row) {
                 Object.keys(row).forEach(function (key) {
-                    var currentColumn = row[key].text,
+                    var currentColumn,
+                        currentColumnLength,
+                        largestColumn,
+                        largestColumnLength;
+                    if (!row[key]){
+                        //do nothing, no value for this column;
```

Comment:
Hrm; this should fail to pass JSLint as it's an empty block.  Can you invert the truth test and remove the empty block?